LM IR BLinker.X/main.c

```diff
@@ -43,9 +43,21 @@
 
 #include "mcc_generated_files/mcc.h"
 
-/*
-                         Main application
- */
+// ========== DEFINES ==========
+#define LED_PIN     LATAbits.LATA3
+#define PWM_EN      PWM3CONbits.EN
+ 
+
+// ========== GLOBAL VARIABLES ==========
+const bool pattern[] = {0, 1, 0, 1, 1, 1};

```

Comment:
This pattern can be any length. Just add more zeros and ones in there. The variable type is "bool", so if you try to stick a "2" or something in there the compiler won't let you flash it. "const" will stick these values in non-volatile memory (you have more of that than you have RAM, so it's good to do that where you can)